package.json

```diff
@@ -151,14 +161,17 @@
     "nightwatch_remote_parallel": "nightwatch --config nightwatch.js --env ie,safari,chrome,default",
     "nightwatch_remote_safari": "nightwatch --config nightwatch.js --env safari",
     "onchange": "onchange build/app.js -- npm-run-all lint",
-    "prepublish": "mkdirp build; npm-run-all -ls downloadsolc build",
+    "prepublish": "mkdirp build; npm-run-all -ls downloadsolc make-mock-compiler build",
```

Comment:
I checkout to master, pulled (there was nothing new), back to righthand and then git rebase master and I got this conflicts

```js
<<<<<<< HEAD
  function swarmVerifiedPublish (content, expectedHash, cb) {
    swarmgw.put(content, function (err, ret) {
      if (err) {
        cb(err)
      } else if (ret !== expectedHash) {
        cb('Hash mismatch')
      } else {
        cb()
      }
    })
  }

  function publishOnSwarm (contract, cb) {
    // gather list of files to publish
    var sources = []

    sources.push({
      content: contract.metadata,
      hash: contract.metadataHash
    })

    var metadata
    try {
      metadata = JSON.parse(contract.metadata)
    } catch (e) {
      return cb(e)
    }

    if (metadata === undefined) {
      return cb('No metadata')
    }

    Object.keys(metadata.sources).forEach(function (fileName) {
      // find hash
      var hash
      try {
        hash = metadata.sources[fileName].urls[0].match('bzzr://(.+)')[1]
      } catch (e) {
        return cb('Metadata inconsistency')
      }

      sources.push({
        content: files.get(fileName),
        hash: hash
      })
    })

    // publish the list of sources in order, fail if any failed
    async.eachSeries(sources, function (item, cb) {
      swarmVerifiedPublish(item.content, item.hash, cb)
    }, cb)
  }

  udapp.event.register('publishContract', this, function (contract) {
    publishOnSwarm(contract, function (err) {
      if (err) {
        alert('Failed to publish metadata: ' + err)
      } else {
        alert('Metadata published successfully')
      }
    })
=======
  udapp.event.register('publishContract', this, function (contract) {
    console.log('Publish contract...', contract)
    alert('Publishing a contract is not supported yet.')
>>>>>>> Add dummy publishContract listener
  })
```